tests/test_browser.py

```diff
@@ -4840,9 +4835,6 @@ def test_browser_modularize_no_current_script(self):
       ''' % creation)
       self.run_browser('/'.join(path + ['test.html']), None, '/report_result?0')
 
-  def test_modularize_Module_input(self):
-    self.btest(path_from_root('tests', 'browser', 'modularize_Module_input.cpp'), '0', args=['--shell-file', path_from_root('tests', 'browser', 'modularize_Module_input.html'), '-s', 'MODULARIZE_INSTANCE=1'])
```

Comment:
Alright, I'll add this back

Reply:
I believe `test_modularize` already covers testing the behavior of passing in `Module` defaults to the factory function. I decided to delete this one because looking at the HTML/script fixture, this test looked to be explicitly testing the behavior where `MODULARIZE_INSTANCE` would use a global `Module` variable if it was defined. I'm not sure how I would adjust the test to still be useful if I didn't delete it.

Reply:
I see now, yeah, sounds good.